test/remote/gateways/remote_adyen_test.rb

```diff
@@ -1485,6 +1485,20 @@ def test_successful_authorize_with_sub_merchant_sub_seller_data
     assert_success response
   end
 
+  def test_sending_mcc_on_authorize
+    options = {
+      reference: '345123',
+      email: 'john.smith@test.com',
+      ip: '77.110.174.153',
+      shopper_reference: 'John Smith',
+      order_id: '123',
+      mcc: '5411'
+    }
+    response = @gateway.authorize(@amount, @credit_card, options)
+    assert_failure response
+    assert_equal 'Could not find an acquirer account for the provided currency (USD).', response.message
```

Comment:
I would have liked to write a successful test for this, but it appears that the test account has no associated acquirer accounts for...any currency. And evidently when you send the MCC field that is required. This will be something that the merchant has to configure in order to send this field.